resources/views/community/components/message/new-thread-page.blade.php

```diff
@@ -0,0 +1,56 @@
+@props([
+    'toUser' => '',
+    'subject' => '',
+    'message' => '',
+])
+
+<x-app-layout
+    pageTitle="New Message"
+    pageDescription="Create a new message"
+>
+    <x-user.breadcrumbs targetUsername="{{ request()->user()->User }}"
+            parentPage="Messages"
+            parentPageUrl="{{ route('message.list') }}"
+            currentPage="New Message" />
+
+    <div class="w-full flex gap-x-3">
+        <h1 class="mt-[10px] w-full">New Message</h1>
+    </div>
+
+    <form action='/request/message/create.php' method='post' x-data='{ isValid: true }'>
+        {{ csrf_field() }}
+        <div><table class='mb-4'><tbody>
+            <tr>
+                <td><label for='recipient'>User:</label></td>
+                <td>
+                    <div class="w-full">
+                        <div style="float:right">
+                            <x-input.user-select-image for="recipient" :user="$toUser" size="48" />
+                        </div>
+                        <!-- TODO: why won't this field align properly? -->
+                        <div style='vertical-align:middle'>
+                            <x-input.user-select name="recipient" :user="$toUser" />
+                        </div>
+                    </div>
```

Comment:
Floats can make it tricky. I recommend always reaching for flex before trying to float an element. This should resolve the issue outright or get you very close:

```html
<div class="w-full flex justify-between items-center">
  <div><x-input.user-select ... /></div>
  <div><x-input.user-select-image ... /></div>
</div>
```

Note that I reversed the order of the inputs so the flex layout can set them correctly.

In a flex layout, `justify-*` controls element alignment along the X-axis and `items-*` controls element alignment along the Y-axis.

`justify-between` forces the elements along the X-axis to be "pushed" as far away as possible from each other.
`items-center` forces the elements along the Y-axis to be horizontally centered.

This is a quick and somewhat silly resource for how to master flex: https://flexboxfroggy.com/

Reply:
Thanks.